src/views/project/iam/ProjectRoleTab.vue

```diff
@@ -61,7 +61,12 @@
             </a-tooltip>
           </span>
         </a-table>
-        <a-modal title="Edit Project Role" v-model="editModalVisible" :footer="null" :afterClose="closeAction">
+        <a-modal
+          title="Edit Project Role"
```

Comment:
@Pearl1594 I think these are your feature - can you send another PR that fixes the translation keys?